package.json

```diff
@@ -19,9 +19,11 @@
     "build-tokens": "paragon build-tokens --source ./tokens/src --build-dir ./paragon/css --source-tokens-only",
     "build-scss": "paragon build-scss --corePath ./paragon/core.scss --themesPath ./paragon/css/themes --defaultThemeVariants light",
     "build": "make build",
+    "build:watch": "nodemon --ignore dist -x \"make build\"",
```

Comment:
Running `npm run build:watch` will automatically re-build the tokens/theme, resulting in updated `dist` output on updates to files. Useful when referencing the locally built files from Paragon during local development, or just more streamlined rapid development.